source/PyMaterialX/PyMaterialXRender/PyImage.cpp

```diff
@@ -10,11 +10,18 @@
 namespace py = pybind11;
 namespace mx = MaterialX;
 
+uintptr_t getResourceBuffer(const mx::Image& image)
```

Comment:
Cannot pass back and forth void pointers for buffer data. 
Add a concrete type to wrap.